.github/workflows/examples.yml

```diff
@@ -37,9 +37,9 @@ jobs:
       run: |
         # TODO(hvy): Stop ignoring allennlp_jsonnet.py when https://github.com/optuna/optuna/issues/1526 is resolved.
         if [ ${{ matrix.python-version }} = 3.5 ]; then
-          IGNORES='chainermn_.*|pytorch_lightning.*|fastai_.*|allennlp_.*|catalyst_.*|sb3_.*|allennlp_jsonnet.py'
+          IGNORES='chainermn_.*|pytorch_lightning.*|fastai_.*|allennlp|catalyst_.*|sb3_.*|allennlp_jsonnet.py'
```

Comment:
[just a question]

We can remove `allennlp_jsonnet.py` instead? (same for L42)

Reply:
Fixed!